### PR TITLE
Bind/unbind stream channels from the definition

### PR DESCRIPTION
**Pull request by zazabe**

### CODE REVIEW

**Comment by zazabe** [comment]
@vogdb please review (with love)

**Comment by vogdb** [comment]
lgtm

File: library/CM/StreamChannel/Definition.js [hunk 1]
```diff
@@ -18,5 +18,24 @@ var CM_StreamChannel_Definition = CM_Frontend_JsonSerializable.extend({
...
     return this.get('type');
+  },
+
+  /**
+   * @param {String} eventName
+   * @param {Function} callback fn(array data)
+   * @param {Object} [context]
+   * @param {Boolean} [allowClientMessage]
+   */
+  bind: function(eventName, callback, context, allowClientMessage) {
```
Comment (tomaszdurka) [inline library/CM/StreamChannel/Definition.js]: Spynote: Naming might be bit too general.

Comment (zazabe) [inline library/CM/StreamChannel/Definition.js]: agree > https://github.com/cargomedia/cm/pull/2254

**Pull request merged**